apollo-federation/src/merge.rs

```diff
@@ -1088,6 +1105,13 @@ fn join_field_directive_definition() -> DirectiveDefinition {
                 ty: ty!(String).into(),
                 default_value: None,
             }),
+            Node::new(InputValueDefinition {
+                name: name!("overrideLabel"),
```

Comment:
Same Q re: const

Reply:
it's a fair question but this whole file is filled with strings and i don't really want to make a bigger diff than necessary! i'll replace this one at least